app/src/main/java/com/teamagam/gimelgimel/app/mainActivity/view/MainActivityPanel.java

```diff
@@ -73,6 +74,8 @@ public void onResume() {
         super.onResume();
         mSlidingLayout.addPanelSlideListener(mPanelListener);
         mBottomViewPager.addOnPageChangeListener(mPageListener);
+        mSlidingPanelOffset = mSlidingLayout.getCurrentSlideOffset();
+        getViewTreeObserver().addOnGlobalLayoutListener(createGlobalLayoutListener());
```

Comment:
Take a look at the listener itself.
The listener removes itself after doing its work.